src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -596,20 +599,24 @@ describe("DatePicker", () => {
       // Helper text for the "To" input
       expect(screen.getByText(/Note for the 'to' field./i)).toBeInTheDocument();
 
-      const fromInput = screen.getByRole("textbox", { name: "From" });
-      const toInput = screen.getByRole("textbox", { name: "To" });
+      const inputFrom = screen.getByRole("textbox", {
+        name: "From - Note for the 'from' field., Press tab to access the calendar.",
```

Comment:
The extra comma after the period is odd. It's not really an issue.

Reply:
I agree it's a bit weird, but the reason I added a comma is that there is not necessary a period after the footnote. In this case, the `aria-label` might look like this, "From - Note for the 'from' field Press tab to access the calendar." Adding the comma ensures the screen reader pauses before reading the "Press tab..." piece. Is there another way to do this? Or do you feel it's not important that the screen reader pause between these two pieces of information?

Reply:
You could read the last character in the string and check for a period, but tbh it seems like more effort than is needed. The premeditated comma seems fine. It'd be good to see how others feel about it.